packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1233,11 +1241,11 @@ class SemanticsOwner extends ChangeNotifier {
   ///
   /// If the [SemanticsNode] has not indicated that it can perform the action,
   /// this function does nothing.
-  void performAction(int id, SemanticsAction action) {
+  void performAction(int id, SemanticsAction action, [dynamic args]) {
```

Comment:
I'm wondering if making this method generic (i.e. `T args`) would buy us anything?

Reply:
Discussed offline: Doesn't buy us anything.